php-practice/section12.php

```diff
@@ -64,6 +64,28 @@ public function getBar() {
  * In the child class, access the parent's protected properties and methods.
  */
 
+class BlackSheep {
+  protected $sheeple = 'People but sheep (iz a joke)';
+
+  protected function punchLine() {
+    echo 'Imagine all the sheeple!' . PHP_EOL;
+  }
+}
+
+class LittleSheeple extends BlackSheep {
+  public function callPunchLine() {
+    $this->punchLine();
+  }
+
+  public function getSheeple() {
+    return $this->sheeple;
+  }
+}
+
+$littleSheepleObject = new LittleSheeple();
+$littleSheepleObject->callPunchLine();
+echo $littleSheepleObject->getSheeple();
```

Comment:
FYI, calling this method already echoes out to the console. Echoing out the return value of the function doesn't do anything because the function doesn't return anything.